README.md

```diff
@@ -1,7 +1,128 @@
-#call
-
-HTTP Router.
+# call - Simple HTTP Router
 
 [![Build Status](https://secure.travis-ci.org/hapijs/call.png)](http://travis-ci.org/hapijs/call)
 
-Lead Maintainer - [Eran Hammer](https://github.com/hueniverse)
+### Lead Maintainer - [Eran Hammer](https://github.com/hueniverse)
+
+## Introduction
+```call``` is a simple node.js HTTP Router. It is used by popular [hapi.js](https://github.com/hapijs/hapi) web framework. It implements predictable and easy to use routing. Even if it is designed to work with Hapi.js, you can still use it as independent component.
+
+## Getting Started
+
+### Installation
```

Comment:
Other hapi docs don't have this and I don't think we care how you install this. Also why as a dev dep?